cipher/rijndael.c

```diff
@@ -1769,6 +1769,146 @@ selftest (void)
   return r;
 }
 
+/* XTS vectors copied from GNUTLS. */
```

Comment:
Do we need to worry about the GnuTLS license?

Reply:
The only thing copied is test vectors, I don't believe they can be covered under copyleft licensing.